.github/workflows/master.yml

```diff
@@ -0,0 +1,27 @@
+name: MLflow tests
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  python-small:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python 3.6
+      uses: actions/setup-python@v1
+      with:
+        python-version: 3.6
+    - name: Install dependencies
+      run: |
+        export GITHUB_WORKFLOW=1
+        INSTALL_SMALL_PYTHON_DEPS=true source ./travis/install-common-deps.sh
+    - name: Run tests
+      run: |
+        export GITHUB_WORKFLOW=1
+        export PATH="$HOME/miniconda/bin:$PATH"
```

Comment:
Had to re-export and re-activate the test environment, since these are set in `install-common-deps.sh` but Github Actions clears env variables between steps.